climate/vicare.py

```diff
@@ -149,6 +168,34 @@ def turn_away_mode_off(self):
         self.schedule_update_ha_state()
         self._api.setMode("normal")
 
+    def set_hold_mode(self, hold_mode):
+        if hold_mode in ["away", "home"]:
+            active_mode = self._api.getActiveMode()
+            self.schedule_update_ha_state()
+            if hold_mode == "away":
+                success = self._api.setMode("forcedReduced")
+            else:
+                hold_mode == "home"
+                success = self._api.setMode("forcedNormal")
+            # PyViCare currently returns 'None' on no error, checking for both for future changes
+            if success["error"] is None or success["error"] == 'None':
+                self._hold = hold_mode
+                if active_mode in ["standby", "dhw", "dhwAndHeating", "active"]:
```

Comment:
shouldn't we add 'forcedReduced' and 'forcedNormal' here? Otherwise if a user operates his heating on one of those modes permanently "hold mode off" won't switch back to those states.